Chrome 115 - cannot find matching chrome driver

### Orb version

1.4.2, 1.4.3

### What happened

Chrome is being installed:
```
Google Chrome 115.0.5790.98  has been installed to /usr/bin/google-chrome-stable
```

but it is not matched to download proper chrome driver:

```
Installed version of Google Chrome is 115.0.5790.98 
ChromeDriver  is already installed
```

and during tests:
```
Could not start a new session. Response code 500. Message: session not created: This version of ChromeDriver only supports Chrome version 102
Current browser version is 115.0.5790.98 with binary path /usr/bin/google-chrome 
```

### Expected behavior

Proper chrome driver should be installed and used to match browser version

I cannot push to fix/ branch so that's the diff that might help:
```
Index: src/scripts/install-chromedriver.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/scripts/install-chromedriver.sh b/src/scripts/install-chromedriver.sh
--- a/src/scripts/install-chromedriver.sh	(revision 0b7c7237d565b71272f53e20a9449d64dcc46791)
+++ b/src/scripts/install-chromedriver.sh	(revision e51ac0112c2a891e2c9d1fbf509412bd28b11d87)
@@ -125,9 +125,11 @@
     exit 1
     ;;
   esac
-  elif [[ $CHROME_RELEASE -lt 115 ]]; then
+elif [[ $CHROME_RELEASE -lt 115 ]]; then
     CHROMEDRIVER_VERSION=$(curl --silent --show-error --location --fail --retry 3 \
       "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROMEDRIVER_RELEASE")
+else
+   CHROMEDRIVER_VERSION=$(echo $CHROME_VERSION | sed 's/[^0-9.]//g')
 fi
 
 # installation check
@@ -150,7 +152,6 @@
     "http://chromedriver.storage.googleapis.com/$CHROMEDRIVER_VERSION/chromedriver_$PLATFORM.zip"
 else 
    # shellcheck disable=SC2001
-  CHROMEDRIVER_VERSION=$(echo $CHROME_VERSION | sed 's/[^0-9.]//g')
   echo "$CHROMEDRIVER_VERSION will be installed"
   if [[ $PLATFORM == "linux64" ]]; then
     PLATFORM="linux64"
```

In general I've moved `CHROMEDRIVER_VERSION=$(echo $CHROME_VERSION | sed 's/[^0-9.]//g')` to `else` as stated above.

--- a/src/scripts/install-chromedriver.sh
+++ b/src/scripts/install-chromedriver.sh
@@ -128,6 +128,9 @@ if [[ $CHROME_RELEASE -lt 70 ]]; then
   elif [[ $CHROME_RELEASE -lt 115 ]]; then
     CHROMEDRIVER_VERSION=$(curl --silent --show-error --location --fail --retry 3 \
       "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROMEDRIVER_RELEASE")
+  else
+    # shellcheck disable=SC2001
+    CHROMEDRIVER_VERSION=$(echo $CHROME_VERSION | sed 's/[^0-9.]//g')
 fi
 
 # installation check
@@ -141,16 +144,14 @@ if command -v chromedriver >/dev/null 2>&1; then
   fi
 fi
 
-# download chromedriver
+echo "ChromeDriver $CHROMEDRIVER_VERSION will be installed"
 
+# download chromedriver
 if [[ $CHROME_RELEASE -lt 115 ]]; then
-  echo "ChromeDriver $CHROMEDRIVER_VERSION will be installed"
   curl --silent --show-error --location --fail --retry 3 \
     --output chromedriver_$PLATFORM.zip \
     "http://chromedriver.storage.googleapis.com/$CHROMEDRIVER_VERSION/chromedriver_$PLATFORM.zip"
 else 
-   # shellcheck disable=SC2001
-  CHROMEDRIVER_VERSION=$(echo $CHROME_VERSION | sed 's/[^0-9.]//g')
   echo "$CHROMEDRIVER_VERSION will be installed"
   if [[ $PLATFORM == "linux64" ]]; then
     PLATFORM="linux64"

